ROLES.md

```diff
@@ -188,18 +175,19 @@ The following apply to the part of the codebase for which one would be an approv
 
 ## Lead
 
-Working group leads, or just ‘leads’, are approvers of an entire area that have demonstrated good judgement and responsibility.
-Leads accept design proposals and approve design decisions for their area of ownership.
+Working group leads, or just ‘leads’, are maintainers of an entire area that have demonstrated good judgement and responsibility.
```

Comment:
For TOC discussion:  the implication is that working group leads who no longer meet the recent contribution bar of being a maintainer can no longer be working group leads.

An alternative that could be considered:  working group leads who no longer meet the maintainer bar can continue to be facilitators but lose their voting rights (depends on the suggestion above that maintainers get a binding vote on design reviews/feature proposals)

Reply:
I think it is useful for us to start making votes with binding and non-binding.  I find out one challenge with the project is that we can't make decision easily, lots of voices and there is no clear process in place for voting which we absolutely need.